### PR TITLE
Added Foundation Calendar info +formatting updates

### PR DESCRIPTION
**Pull request by williamkapke**
Also updated:
- corrected my github handle
- sorted "contacts" A-Z
- normalized handle display

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,15 +24,37 @@ If an agenda item cannot reach a consensus a member can call for either a closin
...
+- Matteo Collina [@mcollina](https://github.com/mcollina)
+- Michael Dawson [@mhdawson](https://github.com/mhdawson)
+- Rich Trott [@trott](https://github.com/trott)
+- Ryan Lewis [@ryanmurakami](https://github.com/ryanmurakami)
+- Tracy Hinds [@hackygolucky](https://github.com/hackygolucky)
+- William Kapke [@williamkapke](https://github.com/williamkapke)
+
+All calendar maintainers have `Make changes AND manage sharing` permissions. If you would like to help maintain your team's calendar events, open a PR adding your name to the list above. Once approved, one of the calendar maintainers will add you to the calendar settings.
+
+This list should be reviewed and pruned annually (at minimum). The calendar has a yearly recurring event on Jan 31st for this. An issue should be opened asking the calendar maintainers for their continued volunteering efforts (directly @-mention all members). After 1 week, the this list should be PRed removing members that did not respond. [The calendar permissions to be updated once merged](https://support.google.com/a/answer/117596?hl=en).
```
Comment (targos) [inline README.md]: the this -> this

**Comment by Trott** [comment]
Not a blocker, but I would recommend sorting names on GitHub handle rather than name. The reason is that the GitHub handle will be a limited number of easy-to-sort characters. If you sort on name, you may find yourself required to figure out where Сковорода Никита Андреевич or 顾天骋 goes in the list.

**Comment by williamkapke** [comment]
Good call @Trott. You're very wise. Updated.

File: README.md [hunk 2]
```diff
@@ -43,15 +43,16 @@ https://nodejs.org/calendar
 <sup>Click `+GoogleCalendar` at the bottom right to add to your own Google calendar.</sup>
 
 The calendar is maintained by:
-- Bryan Hughes [@nebrius](https://github.com/nebrius)
+<!-- sorted by GitHub handle -->
```
Comment (gibfahn) [inline README.md]: Optional, but if you put the Github ID first, as in the [nodejs/node README](https://github.com/nodejs/node/#tsc-technical-steering-committee), the ordering should be more clear.

Comment (williamkapke) [reply README.md]: Cool. I was leaning that way in my head but didn't know of any other list displayed like that. Updated now.

**Comment by mhdawson** [comment]
Landed as 00197817151b49ff6d9f7c9f0f6a62ae25e66237

**Pull request closed without merging**